src/cryptsetup/cryptsetup-generator.c

```diff
@@ -227,7 +227,7 @@ static int create_disk(
                 *filtered = NULL, *u_escaped = NULL, *filtered_escaped = NULL, *name_escaped = NULL, *header_path = NULL;
         _cleanup_fclose_ FILE *f = NULL;
         const char *dmname;
-        bool noauto, nofail, tmp, swap, netdev;
+        bool noauto, nofail, tmp, swap, netdev, create_in_initrd;
```

Comment:
i think adding this mount option mirroring x-initrd.mount makes a lot of sense. But I'd really cal this "x-initrd.attach" or so, after all we usually talk about attaching LUKS volumes, not "creating" them. (i.e. creating a LUKS volume probably would mostly understand as "formatting something as LUKS")